src/tscore/ParseRules.cc

```diff
@@ -75,7 +75,7 @@ ink_atoi64(const char *str)
     while (*str && ParseRules::is_digit(*str)) {
       num = (num * 10) - (*str++ - '0');
     }
-#if USE_SI_MULTILIERS
+#if USE_SI_MULTIPLIERS
     if (*str) {
```

Comment:
The same as #5381 you should fix the definition in ParseRules.h at the same time .